math/hammersley.glsl

```diff
@@ -0,0 +1,32 @@
+#include "const.glsl"
+
+/*
+contributors: Holger Dammertz
+description: Return a Hammersley point
+use: hammersley(uint index)
+license: CC BY 3.0 Copyright (c) 2012 Holger Dammertz
+*/
+
+#if !defined(FNC_HAMMERSLEY) &&  __VERSION__ >= 130
+#define FNC_HAMMERSLEY
+
+vec2 hammersley(uint index, int numSamples) {
+    const float tof = 0.5 / float(0x80000000U);
+    uint bits = index;
+    bits = (bits << 16u) | (bits >> 16u);
+    bits = ((bits & 0x55555555u) << 1u) | ((bits & 0xAAAAAAAAu) >> 1u);
+    bits = ((bits & 0x33333333u) << 2u) | ((bits & 0xCCCCCCCCu) >> 2u);
+    bits = ((bits & 0x0F0F0F0Fu) << 4u) | ((bits & 0xF0F0F0F0u) >> 4u);
+    bits = ((bits & 0x00FF00FFu) << 8u) | ((bits & 0xFF00FF00u) >> 8u);
+    return vec2(float(index) / numSamples, float(bits) * tof);
+}
+
```

Comment:
```
vec2 hammersley(int index, int numSamples) { return hammersley(uint(index), numSamples); }
```

Would be great to have a version for __VERSION__ < 130

Reply:
I'll put it on my list ;-)